scripts/migrate_registration_and_fork_log.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+import sys
+import logging
+
+from website.app import init_app
+from website.models import NodeLog, Node
+from scripts import utils as script_utils
+from modularodm import Q
+
+logger = logging.getLogger(__name__)
+
+
+def get_all_parents(node):
+    # return a list contains all possible forked_from and registered_from of the node to the very origin
+    parent_list = []
+    while True:
+        if get_parent(node) is None:
```

Comment:
In order to get all parents for the current node, since it can be nested fork and the current node should have the log from all of its fork parent.